install/package.rst

```diff
@@ -31,6 +31,36 @@ some operating systems may require additional packages if not already installed.
 
          $ yum install wget epel-release
 
+         # CentOS 7
+         $ yum install https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm
+
+(optional) Node.js
+~~~~~~~~~~~~~~~~~~
+
+Zammad requires Node.js for compiling it's assets.
+Package installations by default come shipped with assets compiled already.
+
+You'll only have to install this package in case you're going to change
```

Comment:
to install Node.js in case (reference seems unclear as package based installs mentioned in the previous sentence)

Reply:
Good point, I exchanged ``this package`` with ``node.js``